src/components/DateTimePicker/Internal/Utils/uiUtil.ts

```diff
@@ -220,25 +231,23 @@ let globalClickFunc: ClickEventHandler | null = null;
 const clickCallbacks: Set<ClickEventHandler> = new Set<ClickEventHandler>();
 
 export function addGlobalMouseDownEvent(callback: ClickEventHandler) {
-  if (
-    !globalClickFunc &&
-    typeof window !== 'undefined' &&
-    window.addEventListener
-  ) {
+  if (!globalClickFunc && canUseDom()) {
     globalClickFunc = (e: MouseEvent) => {
       // Clone a new list to avoid repeat trigger events
       [...clickCallbacks].forEach((queueFunc) => {
         queueFunc(e);
       });
     };
-    window.addEventListener('mousedown', globalClickFunc);
+    if (window.addEventListener) {
+      window.addEventListener('mousedown', globalClickFunc);
+    }
   }
 
   clickCallbacks.add(callback);
```

Comment:
without the listener, is there value in the callback? or will it get added after canUseDom?

Reply:
We useEffect on this function, so it should be added after the DOM is available.

Reply:
To answer your question, the callback has value in that it's stored for later use, but it will only be used if the DOM becomes available at some point and the mousedown event is triggered.